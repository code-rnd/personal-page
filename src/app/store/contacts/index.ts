export { contactsActions } from "./contacts.actions";
export { contactsReducer } from "./contacts.reducer";
export * from "./contacts.selector";
