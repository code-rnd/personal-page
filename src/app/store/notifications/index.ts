export { notificationsActions } from "./notifications.actions";
export { notificationsReducer } from "./notifications.reducer";
export * from "./notifications.selector";
