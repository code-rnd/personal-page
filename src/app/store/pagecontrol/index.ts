export { pageControlActions } from "./pagecontrol.actions";
export { pagecontrolReducer } from "./pagecontrol.reducer";
export * from "./pagecontrol.selector";
