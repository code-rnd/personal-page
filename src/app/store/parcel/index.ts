export { parcelActions } from "./parcel.actions";
export { parcelReducer } from "./parcel.reducer";
export * from "./parcel.selector";
