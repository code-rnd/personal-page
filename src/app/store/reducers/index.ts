import { combineReducers } from "redux";

import { historyReducer as history } from "../history";
import { authorizationReducer as authorization } from "../authorization";
import { routeReducer as route } from "../route";
import { pagecontrolReducer as pagecontrol } from "../pagecontrol";
import { contactsReducer as contacts } from "../contacts";
import { notificationsReducer as notifications } from "../notifications";

export const reducers = combineReducers({
  history,
  authorization,
  route,
  pagecontrol,
  contacts,
  notifications,
});
