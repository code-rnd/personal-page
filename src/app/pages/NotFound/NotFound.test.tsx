import React from "react";
import renderer from "react-test-renderer";
import { Provider } from "react-redux";

import { NotFound } from "./index";
import { store } from "../../store";

it(`Компонент ${NotFound.name}, отрисован корректно`, () => {
  const Page = renderer
    .create(
      <Provider store={store}>
        <NotFound />
      </Provider>
    )
    .toJSON();

  expect(Page).toMatchSnapshot();
});
