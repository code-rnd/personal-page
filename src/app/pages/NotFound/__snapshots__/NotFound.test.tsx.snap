// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Компонент undefined, отрисован корректно 1`] = `
<div
  className="page-body"
>
  <div
    className=""
  >
    404 Not found
  </div>
</div>
`;
